<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewProperty</class>
 <widget class="QDialog" name="AddNewProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_propertyDetails">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Add New Property:</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>491</width>
      <height>287</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_nickname"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_address">
         <property name="text">
          <string>Address</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_address</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_address"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_purchaseCost">
         <property name="text">
          <string>Purchase Cost</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_monthlyMortgage">
         <property name="text">
          <string>Monthly Mortgage</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_monthlyMortgage_2">
         <property name="text">
          <string>Mortgage Term in Years</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_HOA">
         <property name="text">
          <string>HOA fee</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_sewage">
         <property name="text">
          <string>Sewage Company</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_sewage</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_sewage"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_utility">
         <property name="text">
          <string>Utility Company</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_utility</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEdit_utility"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_notes_2">
         <property name="text">
          <string>Notes</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_notes</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="lineEdit_notes"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_nickname">
         <property name="text">
          <string>Property Nickname</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_address</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_purchaseDate">
         <property name="text">
          <string>Date of Purchase:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="datePurchase">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinMonthlyMortgage">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>500.000000000000000</double>
         </property>
         <property name="value">
          <double>2500.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinMortgageYears">
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinHOA">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>10.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinPurchaseCost">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>500.000000000000000</double>
         </property>
         <property name="value">
          <double>2500000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushSavePropertyDetails">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
